### PR TITLE
Adding FuzzyInference Algorithm, with Accord.Fuzzy reference to Project

### PR DESCRIPTION
**Pull request by espirulina**

### CODE REVIEW

File: Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj [hunk 1]
```diff
@@ -12,6 +12,8 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <LangVersion>5</LangVersion>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```
Comment (mchandschuh) [inline Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj]: Is this required?

File: Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj [hunk 2]
```diff
@@ -135,6 +154,13 @@
     <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="..\packages\Accord.3.0.2\build\Accord.targets" Condition="Exists('..\packages\Accord.3.0.2\build\Accord.targets')" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
+    <PropertyGroup>
+      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
```
Comment (mchandschuh) [inline Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj]: Shouldn't need to add this build target.

File: Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj (hunk 1 above)
Comment (espirulina) [inline Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj]: I dont think so. I think it was added automatically upon adding the NuGet package

File: Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj (hunk 2 above)
Comment (espirulina) [inline Algorithm.CSharp/QuantConnect.Algorithm.CSharp.csproj]: Indeed

**Pull request merged**